setup.py

```diff
@@ -0,0 +1,32 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="asaplib",
```

Comment:
Is there a difference between installing `asap` as a python package viz-a-viz `asaplib`?

Reply:
the name `asap` is taken on pypi as well, so using the already used inner one `asaplib` (not taken on pypi as of a few days ago when I checked) would perhaps be more straightforward. Also enables us to publish the package and enable `pip install asaplib` as an easy way